tools/cli.js

```diff
@@ -50,26 +104,36 @@ rl.on('close', () => {
   process.exit();
 });
 
-function completer(line) {
-  return [[], line];
-}
-
 const state = {
   client: null,
   connection: null
 };
 
+commandProcessor.complete = (inputs, depth) => {
+  const completions = ['call', 'event', 'connect', 'disconnect', 'exit'];
```

Comment:
I'd prefer these to be sorted alphabetically, though I don't feel strongly for it.